core-bundle/src/Altcha/Altcha.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Altcha;
+
+use Contao\CoreBundle\Altcha\Config\Algorithm;
+use Contao\CoreBundle\Altcha\Exception\InvalidAlgorithmException;
+use Contao\CoreBundle\Entity\Altcha as AltchaEntity;
```

Comment:
`AltchaEntity` is an alias for `\Contao\CoreBundle\Entity\Altcha`

Reply:
Ah, right, sorry.